source/standalone/workflows/skrl/play.py

```diff
@@ -26,6 +26,14 @@
 parser.add_argument("--num_envs", type=int, default=None, help="Number of environments to simulate.")
 parser.add_argument("--task", type=str, default=None, help="Name of the task.")
 parser.add_argument("--checkpoint", type=str, default=None, help="Path to model checkpoint.")
+parser.add_argument(
+    "--ml_framework",
+    type=str,
+    default="torch",
+    choices=["torch", "jax", "jax-numpy"],
+    help="ML framework used for training.",
```

Comment:
Renaming for clarity that this is for the agent side and not the environment side?

```suggestion
    help="The ML framework used for training the SKRL agent.",
```